src/commands/dev/index.ts

```diff
@@ -50,6 +50,12 @@ export default async function main(ctx: NowContext) {
     });
     args = getSubcommand(argv._.slice(1), COMMAND_CONFIG).args;
     output = createOutput({ debug: argv['--debug'] });
+
+    if (argv['--debug']) {
+      process.env.NOW_BUILDER_DEBUG = '1';
+    } else {
+      process.env.NOW_BUILDER_DEBUG = '';
```

Comment:
Is it required to reset it?

Reply:
@AndyBitz initially I thought if user ran with `--debug` it will persist to user environment variable(similar to `export`). turn out it isn't the case. 

thank for pointing it out 🙌